src/observe-tag.js

```diff
@@ -35,6 +54,24 @@ module.exports = tagFunction => {
 		if (oldTag) {
 			oldTag.replaceWith(tagResult)
 
+			// if an element had focus, reapply it
+			if (removedElementWithFocusData.index >= 0) {
+				const children = tagResult.querySelectorAll('*')
+
+				const elementToGiveFocus = [tagResult, ...children][removedElementWithFocusData.index]
```

Comment:
It's creating this array expensive? Should we do a check for 0 and return the tagResult otherwise index +1 on the children array? Or is this operation not frequently called/ not expensive?

Reply:
:thinking: potentially... it could get expensive if the element being re-rendered has like, a ton of children. This part luckily only gets called if the element (or one of it's children) has focus. 

I need to do more generic performance testing, and this could be one of the things that crops up. Without having that performance testing now, it's hard to say for sure.